application/src/test/java/org/opentripplanner/routing/algorithm/filterchain/filters/transit/RemoveTransitIfStreetOnlyIsBetterTest.java

```diff
@@ -17,6 +18,8 @@
 
 public class RemoveTransitIfStreetOnlyIsBetterTest implements PlanTestConstants {
 
+  OptionalInt subscribeResult = OptionalInt.empty();
```

Comment:
Optional as a field here.

Reply:
I created a `RemoveTransitIfStreetOnlyIsBetterResults` record and removed optionals from fields